packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -2151,6 +2246,38 @@ Activate a U2F factor by verifying the registration data and client data.
 
 ##### Activate U2F Request Parameters
 
+| Parameter         | Description                                                | Param Type | DataType | Required |
+| ----------------- | ---------------------------------------------------------- | ---------- | -------- | -------- |
+| factorId          | `id` of factor returned from enrollment                    | URL        | String   | TRUE     |
+| stateToken        | [state token](#state-token) for current transaction        | Body       | String   | TRUE     |
+| registrationData  | base64 encoded registration data from U2F javascript call  | Body       | String   | TRUE     |
+| clientData        | base64 encoded client data from U2F javascript call        | Body       | String   | TRUE     |
```

Comment:
adding the word the and hyphenating base64 encoded

base64-encoded client data from the U2F javascript call